blockchain/proxy_test.go

```diff
@@ -29,6 +33,52 @@ func (m *mockHTTPClient) Do(req *http.Request) (*http.Response, error) {
 	}, nil
 }
 
+func TestElrondProxy_GetHTTPContextDone(t *testing.T) {
+	t.Parallel()
+
+	testHttpServer := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		// simulating that the operation takes a lot of time
+
+		time.Sleep(time.Second * 10)
```

Comment:
can we lower these? the test would take too much to run (1 second, which is the timeout, but still)

Reply:
done